src/components/page-environment/environment-overview.js

```diff
@@ -163,7 +163,7 @@ export class EnvironmentOverview extends Component {
                     ) : (
                       <Typography>Not automatically deployed</Typography>
                     )}
-                    {deployment.gitCommitHash && (
+                    {deployment && deployment.gitCommitHash && (
```

Comment:
same check, just shorter
```suggestion
                    {deployment?.gitCommitHash && (
```